docs/DeveloperGuide.md

```diff
@@ -374,11 +371,79 @@ otherwise)
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. ClickConnect shows an error message.
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**System**: ClickConnect
+
+**Use case**: UC02 - See usage instructions
+
+**Actor**: User
+
+**MSS**
+
+1. User opens ClickConnect application
+2. User selects option to view usage instructions from main menu
+3. ClickConnect displays the usage instructions such as how to add, edit, delete or search for contacts
+4. User reads the instructions to understand how to use the ClickConnect
+5. Use case ends
+
+**Extensions**
+
+* 2a. The application does not have a direct link to usage instructions visible:
```

Comment:
Hmm would this be a bug instead? Since step 2 is "User selects option...", shouldn't the help menu be shown to the user once they request it?